packages/control/lib/runtime-api-client.js

```diff
@@ -17,7 +17,7 @@ class RuntimeApiClient {
   #undiciClients = new Map()
   #webSockets = new Set()
 
-  async getMatchingRuntime (opts) {
+  async getMatchingRuntime (opts = {}) {
```

Comment:
As for the other functions, this will allow us to pass no options at all.

Reply:
Can you add a test for this?